docs/source/markdown/podman-pull.1.md

```diff
@@ -17,12 +21,12 @@ print the full image ID.  **podman pull** can also pull an image
 using its digest **podman pull** *image*@*digest*. **podman pull** can be used to pull
 images from archives and local storage using different transports.
 
-## imageID
-Image stored in local container/storage
+## Image storage
+Images are stored in local image storage.
 
 ## SOURCE
 
- The SOURCE is a location to get container images
+ The SOURCE is the location from which the container images are pulled.
  The Image "SOURCE" uses a "transport":"details" format.
 
  Multiple transports are supported:
```

Comment:
As we're on it, can we remove the list below and point to containers-transports(5) instead? Pull supports all transports and the list below is very likely out-dated.

Reply:
Lets do this in a different PR.  I think this text is repeated in other man pages, and should have active links in markdown.

Reply:
sgtm